common/nodeset-client-v1.go

```diff
@@ -554,12 +556,17 @@ func IsUploadedToNodeset(pubKey beacon.ValidatorPubkey, registeredPubkeys []beac
 	return false
 }
 
-func IsRegisteredToStakewise(pubKey beacon.ValidatorPubkey, statuses map[beacon.ValidatorPubkey]beacon.ValidatorStatus) bool {
-	// TODO: Implement
-	return false
-}
-
-func IsUploadedStakewise(pubKey beacon.ValidatorPubkey, statuses map[beacon.ValidatorPubkey]beacon.ValidatorStatus) bool {
-	// TODO: Implement
-	return false
+func GetNodesetStatus(pubKey beacon.ValidatorPubkey, registeredPubkeysStatusMapping map[beacon.ValidatorPubkey]string) swtypes.NodesetStatus {
+	for registeredPubKey, nodesetStatus := range registeredPubkeysStatusMapping {
+		if registeredPubKey == pubKey {
+			if nodesetStatus == "REGISTERED" {
```

Comment:
Making a note that we'll have to revisit this later to turn these into enum values once they've been solidified

Reply:
done